### ISSUE TITLE
[SF][0.28.1][Table][a11y] Cannot select/deselect a row by pressing space key when using JAWS 2022

### ISSUE THREAD

**Issue by i323808** [SAP/ui5-webcomponents-react#3516]
**Describe the bug**
Cannot select/deselect a row by pressing space key **when using JAWS 2022**.

**Isolated Example**
https://sap.github.io/ui5-webcomponents-react/?path=/docs/data-display-table--default-story

**To Reproduce**
1. Go to https://sap.github.io/ui5-webcomponents-react/?path=/docs/data-display-table--default-story
2. Switch mode to `MultiSelect`
3. **Open JAWS 2022**
4. Press space key to select or deselect row
5. Cannot select or deselect row by pressing space key

**Expected behavior**
User can select or deselect a row by pressing space key when using JAWS.

**UI5 Web Components for React Information**
`@ui5/webcomponents` version: 1.7.1
`@ui5/webcomponents-react` version: 0.28.1
Operating System: Windows 10
Browser: Chrome Version 104.0.5112.102
**Screen Reader: JAWS 2022**

**Comment by Lukas742** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.
